### DISCUSSION TITLE
Customizing the left and right margins

### DISCUSSION THREAD

**Discussion by andersy005** [executablebooks/meta#281] [Q&A] [answered]
Hi 👋🏽

Thank you for such a great sphinx theme (sphinx-book-theme)!

I'd like to reduce the **left** and **right** margins so as to give more screen estate to the main/center container. However, my CSS is limited :). So, I 'm wondering if any CSS experts could help me figure this out? Are these margins customizable via some CSS variables or I need to style specific elements? Any help is approciated. 

![Screen Shot 2021-03-08 at 6 52 48 AM](https://user-images.githubusercontent.com/13301940/110330540-1a418900-7fdb-11eb-90ca-ad3f38b00448.png)

**Comment by choldgraf** [comment, marked as answer]
Hmmm you might be able to get away with something like

```
.container, .container-lg, .container-md, .container-sm, .container-xl {
    max-width: 1600px !important;
}
```

that would define the max width for all container sizes and allow it to get bigger

**Reply by andersy005** [reply]
Thank you, @choldgraf!

**Reply by bitvijays** [reply]
Hey @choldgraf , @andersy005  Hope you are doing well. Could you help with where we would have to make the above change or add the above lines?

We also want to fully utilise the space for all the pages? 

Would the above change will be in `/home/ubuntu/.local/lib/python3.8/site-packages/sphinx_book_theme/static/sphinx-book-theme.css` 

We want to use the [Sphinx Book Theme](https://github.com/executablebooks/sphinx-book-theme)

**Reply by choldgraf** [reply]
You can do so by using Sphinx's functionality for adding custom CSS files: https://docs.readthedocs.io/en/stable/guides/adding-custom-css.html
